.github/CODEOWNERS

```
@@ -3,379 +3,587 @@
+Python/dynload_*.c            @ericsnowcurrently
+
+# Initialisation
 **/*initconfig*               @ericsnowcurrently
 **/*pathconfig*               @ericsnowcurrently
-**/*sysmodule*                @ericsnowcurrently
-**/*bltinmodule*              @ericsnowcurrently
+**/*preconfig*                @ericsnowcurrently
+Doc/library/sys_path_init.rst @FFY00
+Doc/c-api/init_config.rst     @FFY00
+
+# Interpreter main program
+Modules/main.c                @ericsnowcurrently
+Programs/_bootstrap_python.c  @ericsnowcurrently
+Programs/python.c             @ericsnowcurrently
+
+# JIT
+Include/internal/pycore_jit.h @brandtbucher @savannahostrowski @diegorusso
+Python/jit.c                  @brandtbucher @savannahostrowski @diegorusso
+Tools/jit/                    @brandtbucher @savannahostrowski @diegorusso
+
+# Micro-op / μop / Tier 2 Optimiser
+Python/optimizer.c            @markshannon
+Python/optimizer_analysis.c   @markshannon @Fidget-Spinner @tomasr8
+Python/optimizer_bytecodes.c  @markshannon @Fidget-Spinner @tomasr8
+Python/optimizer_symbols.c    @markshannon @tomasr8
+
+# Parser, Lexer, and Grammar
+Grammar/python.gram           @pablogsal @lysnikolaou
+Lib/test/test_peg_generator/  @pablogsal @lysnikolaou
+Lib/test/test_tokenize.py     @pablogsal @lysnikolaou
+Lib/tokenize.py               @pablogsal @lysnikolaou
+Parser/                       @pablogsal @lysnikolaou
+Tools/peg_generator/          @pablogsal @lysnikolaou
+
+# Runtime state/lifecycle
 **/*gil*                      @ericsnowcurrently
-Include/internal/pycore_runtime.h   @ericsnowcurrently
-Include/internal/pycore_interp.h    @ericsnowcurrently
-Include/internal/pycore_tstate.h    @ericsnowcurrently
-Include/internal/pycore_*_state.h   @ericsnowcurrently
+**/*pylifecycle*              @ericsnowcurrently @ZeroIntensity
+**/*pystate*                  @ericsnowcurrently @ZeroIntensity
 Include/internal/pycore_*_init.h    @ericsnowcurrently
+Include/internal/pycore_*_state.h   @ericsnowcurrently
 Include/internal/pycore_atexit.h    @ericsnowcurrently
 Include/internal/pycore_freelist.h  @ericsnowcurrently
 Include/internal/pycore_global_objects.h  @ericsnowcurrently
+Include/internal/pycore_interp.h    @ericsnowcurrently
 Include/internal/pycore_obmalloc.h  @ericsnowcurrently
 Include/internal/pycore_pymem.h     @ericsnowcurrently
+Include/internal/pycore_runtime.h   @ericsnowcurrently
 Include/internal/pycore_stackref.h  @Fidget-Spinner
-Modules/main.c                @ericsnowcurrently
-Programs/_bootstrap_python.c  @ericsnowcurrently
-Programs/python.c             @ericsnowcurrently
+Include/internal/pycore_tstate.h    @ericsnowcurrently
 Tools/build/generate_global_objects.py  @ericsnowcurrently
 
-# Initialization
-Doc/library/sys_path_init.rst @FFY00
-Doc/c-api/init_config.rst     @FFY00
+# Remote Debugging
+Python/remote_debug.h         @pablogsal
+Python/remote_debugging.c     @pablogsal
+Modules/_remote_debugging_module.c @pablogsal @ambv @1st1
```

We have extra indents in some places, but not others. Shall we do it for each block? Even if only for this restructure, and not maintain in updates?

```suggestion
Python/remote_debug.h                 @pablogsal
Python/remote_debugging.c             @pablogsal
Modules/_remote_debugging_module.c    @pablogsal @ambv @1st1
```

---

In general the 'owner' colum is aligned to start at column 31, going back to the initial commit of the file (#2924). Is the suggestion to keep all the entries left-aligned togther in a block where at least one is too long for the "standard" 31 character margin?

---

I've given this a go in [`dae0424` (#137498)](https://github.com/python/cpython/pull/137498/commits/dae0424db50e328fa5866d0e0264b9b6f286bd93), what do you think?